### PR TITLE
T-98 Implement desktop pin functionality

### PR DESCRIPTION
**Pull request by Dr-Frame**

### CODE REVIEW

File: pages/map.js [hunk 1]
```diff
@@ -8,72 +8,16 @@ import Filters from "../components/Filters";
...
-    covering: "штучна трава",
-    opening: "08:00 - 22:00",
-    rating: 4.5,
-    img: image,
-  },
-];
-
 export default function MapPage() {
+  const [childClicked, setChildClicked] = useState(null);
+  const [markerIndex, setMarkerIndex] = useState(0);
```
Comment (WebTarantul) [inline pages/map.js:19]: If sidebar is closed changing of current pin does't opens sidebar

File: components/MapMarkerWrapper/MapMarkerWrapper.js [hunk 2]
```diff
@@ -7,9 +7,18 @@ export default function MapMarkerWrapper({
...
   bgColor,
   className,
+  setMarkerIndex,
+  indexMarker,
 }) {
+  const id = indexMarker;
   return (
-    <div lat={lat} lng={lng} key={key} className={className}>
+    <div
+      onClick={() => setMarkerIndex(id)}
```
Comment (WebTarantul) [inline components/MapMarkerWrapper/MapMarkerWrapper.js:16]: image

File: pages/map.js [hunk 3]
```diff
@@ -8,72 +8,16 @@ import Filters from "../components/Filters";
...
+
   const [sliderOpen, setSliderOpen] = useState(true);
   const [places] = useState(data.courtsDataBase);
   const [filteredPlaces, setFilteredPlaces] = useState([]);
```
Comment (WebTarantul) [inline pages/map.js:23]: useEffect on line 24 is not used. Please remove it

File: components/MapMarkerWrapper/MapMarkerWrapper.js [hunk 4]
```diff
@@ -7,9 +7,18 @@ export default function MapMarkerWrapper({
   typeOfCourt,
   bgColor,
   className,
+  setMarkerIndex,
+  indexMarker,
 }) {
+  const id = indexMarker;
```
Comment (WebTarantul) [inline components/MapMarkerWrapper/MapMarkerWrapper.js:13]: Index is not id

File: components/PlaygroundItem/PlaygroundItem.jsx [hunk 5]
```diff
@@ -4,13 +4,25 @@ import Image from "next/image";
...
+const PlaygroundItem = ({ playground, isActive, handleClick, refProp }) => {
   const playgroundInfoFields = [
     { label: "Тип майданчика", field: "type" },
     { label: "Графік", field: "opening" },
     { label: "Покриття", field: "covering" },
   ];
+
+  const screenWidth = useWindowSize().width;
+
+  if (isActive && screenWidth > 950) {
```
Comment (WebTarantul) [inline components/PlaygroundItem/PlaygroundItem.jsx:18]: This is a some effect then please move that block into useEffect hook with needed depedendsies

File: components/PlaygroundsList/PlaygroundsList.jsx [hunk 6]
```diff
@@ -1,20 +1,43 @@
...
+    setActiveItemId(childClicked);
+  }, [childClicked]);
+
   return (
     <div className={styles.wrapper}>
       <ul className={styles.list}>
-        {playgrounds.map((playground) => (
-          <li className={styles.listItem} key={playground.id}>
+        {playgrounds.map((playground, i) => (
+          <li ref={elRefs[i]} className={styles.listItem} key={playground.id}>
```
Comment (WebTarantul) [inline components/PlaygroundsList/PlaygroundsList.jsx:33]: That way is possible but more clearly if you use forwarding refs(if you want)

https://ru.reactjs.org/docs/forwarding-refs.html

**Review by WebTarantul — CHANGES_REQUESTED** [review]
In general looks good, but I found some moments to fix it.

File: pages/map.js (hunk 1 above)
Comment (Dr-Frame) [reply pages/map.js:19]: I don't fully understand what exactly do you mean

**Review by WebTarantul — APPROVED** [review]

**Comment by Dimirolz** [comment]
@Dr-Frame пофикси конфликты, что бы я смог смерджить

**Pull request merged**